### ISSUE TITLE
[BUG] TFT35 E3 - Unified Menu Material Theme Issue - v26.1

### ISSUE THREAD

**Issue by Neidraidd** [bigtreetech/BIGTREETECH-TouchScreenFirmware#660]
**TLDR: Copy all bmp files from v25 bmp folder into v26.1 bmp folder, DO NOT replacing any files with the same name.**

**Description**
When updating firmware on TFT35 E3 Unified Menu Material Theme is missing bmp files.

**Steps to reproduce**
1. Download v26.1
2. Copy folder "TFT35" and "BIGTREE_TFT35_V3.0_E3.26.1.bin" to SD card
3. Plug SD into screen and boot

**Expected behavior**
All images to display
![IMG_20200507_094932](https://user-images.githubusercontent.com/14808663/81278143-325c1b00-904d-11ea-8e81-1ebf903e5aba.jpg)

**Actual behavior**
Error during install:
Error: Missing images for Tool, Bed and Fan0
![IMG_20200507_094901](https://user-images.githubusercontent.com/14808663/81278125-2b350d00-904d-11ea-8a21-6ed63e7500e5.jpg)

**Hardware Variant**
Ender 3 with SKR v1.4Turbo and TFT35 E3

**TFT Firmware Version & Main Board Firmware details**
Marlin 2.0.x Bugfixes
TFT v26.1

**Additional Information**
None

**Fix**
Copied all bmp files from v25 bmp folder into v26.1 bmp folder, not replacing any files with the same name.

**Comment by WubbuW** [comment]
Just wrong name of the images... remove the underscore from eht 3 status images and will be ok!

**Comment by oldman4U** [comment]
Hello Neidraidd.

I hope you have been able to solve the reported issue in the meantime . So please close this ticket if you do not need it anymore.

Thank you
